components/dashboard/src/admin/Settings.tsx

```diff
@@ -26,12 +43,13 @@ export default function Settings() {
                 <h3>Usage Statistics</h3>
                 <CheckBox
                     title="Enable Service Ping"
-                    desc={<span>This is used to provide insights on how you use your cluster so we can provide a better overall experience. <a className="gp-link" href="https://www.gitpod.io/privacy">Read our Privacy Policy</a></span>}
+                    desc={<span>This is used to provide insights on how you use your Gitpod instance so we can provide a better overall experience. <a className="gp-link" href="https://www.gitpod.io/privacy">Read our Privacy Policy</a>. The following data is sent:</span>}
```

Comment:
Agree with @gtsiolis' suggestions on follow-up improvements for the UI 🔥 Code also looks good to me. I just have a small suggestion for the text. No strong feelings about it, though - can live with the current one :)
```suggestion
                    desc={<span>The following usage data is sent to provide insights on how you use your Gitpod instance, so we can provide a better overall experience. <a className="gp-link" href="https://www.gitpod.io/privacy">Read our Privacy Policy</a>. :</span>}
```

Reply:
Updated!